kokoro/windows/build.bat

```diff
@@ -104,22 +104,36 @@ if "%KOKORO_GITHUB_COMMIT%." == "." (
   set BUILD_SHA=%DEV_PREFIX%%KOKORO_GITHUB_COMMIT%
 )
 
-%BUILD_ROOT%\bazel build -c opt --config symbols ^
+REM Use T:\src for Bazel output, where files are less likely to be
+REM locked by system checks.
+set BAZEL_OUTPUT_USER_ROOT="T:\src\bazel_user_root"
```

Comment:
Use `%BUILD_ROOT%` and name it something short. `out` is already used so why not `%BUILD_ROOT%\build`?

Reply:
done